src/common/DeveloperConfig.ts

```diff
@@ -66,8 +66,6 @@ export type DeveloperConfig = {
   appTileScreens?: AppTileScreens;
   simpleTheme?: SimpleTheme;
   apiBaseURL?: string;
-  navigationTabs?: NavigationTab[];
-  additionalNavigationTabs?: NavigationTab[];
```

Comment:
This is considered a breaking change, removing props.  Definitely not ideal, but I think you & team might be the only one using this atm.  So just expect this to break the build during upgrade.  Down the road, we really need to take a "only add, never remove" approach to these kinds of things 👍